src/sentry/static/sentry/app/stores/configStore.jsx

```diff
@@ -22,15 +22,17 @@ const ConfigStore = Reflux.createStore({
     return this.config;
   },
 
-  loadInitialData(config) {
+  loadInitialData(config, languageOverride = null) {
     config.features = new Set(config.features || []);
     this.config = config;
 
     // TODO(dcramer): abstract this out of ConfigStore
     if (config.user) {
       config.user.permissions = new Set(config.user.permissions);
       moment.tz.setDefault(config.user.options.timezone);
-      setLocale(config.user.options.language || 'en');
+      setLocale(languageOverride || config.user.options.language || 'en');
```

Comment:
How does django determine the language code from `request.LANGUAGE_CODE`? Is there a language supplied when the `lang` query param is not set? Should user's option take precedence?

Reply:
@billyvg uses the same option from the user via our middleware, so ultimately its going to be identical to config.user.options.language